MergerLogicUnitTests/ImageProcessing/TileScalerTest.cs

```diff
@@ -42,6 +42,13 @@ public void BeforeEach()
 
         public static IEnumerable<object[]> GetUpscaleTilesTestParameters()
         {
+            yield return new object[] {
```

Comment:
Could you please add a test where the upscaled tile is fully transparent and is discarded (null tile)?

Reply:
added